app.js

```diff
@@ -3,14 +3,23 @@ const express = require("express");
 const app = express();
 app.use(express.json());
 
-const { getTopics } = require("./new.controllers");
+const { getTopics, getArticleById } = require("./new.controllers");
 
```

Comment:
I would abstract these out into two different controllers/models and put them in folders, otherwise it's going to get really messy.

Reply:
I wouldn't but now i suppose i must